lib/l10n/app_it.arb

```diff
@@ -0,0 +1,600 @@
+  "settingsSubtitleThemeSample": "Questo è un campione",
+  "settingsSubtitleThemeTextAlignmentTile": "Allineamento del testo",
+  "settingsSubtitleThemeTextAlignmentTitle": "Allineamento del testo",
+  "settingsSubtitleThemeTextSize": "Dimensione del testo",
+  "settingsSubtitleThemeShowOutline": "Mostra contorno e ombra",
+  "settingsSubtitleThemeTextColor": "Colore del testo",
+  "settingsSubtitleThemeTextOpacity": "Opacità del testo",
+  "settingsSubtitleThemeBackgroundColor": "Colore di sfondo",
+  "settingsSubtitleThemeBackgroundOpacity": "Opacità dello sfondo",
+  "settingsSubtitleThemeTextAlignmentLeft": "Sinistra",
+  "settingsSubtitleThemeTextAlignmentCenter": "Centro",
+  "settingsSubtitleThemeTextAlignmentRight": "Destra",
+
+  "settingsVideoControlsTile": "Controlli",
+  "settingsVideoControlsTitle": "Controlli",
+  "settingsVideoButtonsTile": "Pulsanti",
+  "settingsVideoButtonsTitle": "Pulsanti",
+  "settingsVideoGestureDoubleTapTogglePlay": "Doppio tocco per play/pausa",
+  "settingsVideoGestureSideDoubleTapSeek": "Doppio tocco sui bordi dello schermo per cercare avanti/indietro",
+
+  "settingsSectionPrivacy": "Privacy",
+  "settingsAllowInstalledAppAccess": "Consentire l'accesso all'inventario delle app",
+  "settingsAllowInstalledAppAccessSubtitle": "Utilizzato per migliorare la visualizzazione degli album",
+  "settingsAllowErrorReporting": "Consenti segnalazione anonima degli errori",
+  "settingsSaveSearchHistory": "Salva la cronologia delle ricerche",
+  "settingsEnableBin": "Usa il cestino",
+  "settingsEnableBinSubtitle": "Conserva gli elementi cancellati per 30 giorni",
+
+  "settingsHiddenItemsTile": "Articoli nascosti",
+  "settingsHiddenItemsTitle": "Articoli nascosti",
+
+  "settingsHiddenFiltersTitle": "Filtri nascosti",
+  "settingsHiddenFiltersBanner": "Le foto e i video che corrispondono ai filtri nascosti non appariranno nella tua collezione",
+  "settingsHiddenFiltersEmpty": "Nessun filtro nascosto",
+
+  "settingsHiddenPathsTitle": "Percorsi nascosti",
+  "settingsHiddenPathsBanner": "Le foto e i video in queste cartelle, o in qualsiasi loro sottocartella, non appariranno nella tua collezione",
+  "addPathTooltip": "Aggiungi percorso",
+
+  "settingsStorageAccessTile": "Accesso a tutti i file",
+  "settingsStorageAccessTitle": "Accesso a tutti i file",
+  "settingsStorageAccessBanner": "Alcune directory richiedono una concessione di accesso esplicita per modificare i file al loro interno. Puoi rivedere qui le directory a cui hai dato accesso in precedenza",
+  "settingsStorageAccessEmpty": "Nessuna autorizzazione concessa",
+  "settingsStorageAccessRevokeTooltip": "Rifiuta autorizzazione",
+
+  "settingsSectionAccessibility": "Accessibilità",
+  "settingsRemoveAnimationsTile": "Rimuovi animazioni",
+  "settingsRemoveAnimationsTitle": "Rimuovi animazioni",
+  "settingsTimeToTakeActionTile": "Tempo per agire",
+  "settingsTimeToTakeActionTitle": "Tempo per agire",
+
+  "settingsSectionDisplay": "Display",
```

Comment:
Unless there's a good reason not to, I'll use `Schermo` for this to be consistent with the Android settings.